src/modules/template.js

```diff
@@ -46,16 +46,16 @@ const PAGELOAD = (() => {
 
 
   return {
-      tabContent,
-      tabs
-  }
+    tabContent,
+    tabs,
+  };
 })();
 
 const TABS = PAGELOAD.tabs;
 const TABCONTENTS = PAGELOAD.tabContent;
```

Comment:
[OPTIONAL] Consider using object destructuring here, if you are trying to follow a certain naming convention, you can rename the variable in the destructuring statement:
```
const {tabs: TABS, tabContent: TABCONTENTS} = PAGELOAD
```

Reply:
Thank you Lucas for the review. Have a great weekend.